composer.json

```diff
@@ -73,11 +73,9 @@
     "scripts": {
         "symfony-scripts": [
             "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
```

Comment:
Nice! However, I would prefer to have a different PR, it's not related to the current one.